WooCommerce/UITestsFoundation/Screens/Settings/BetaFeaturesScreen.swift

```diff
@@ -1,16 +1,24 @@
+import ScreenObject
 import XCTest
 
-class BetaFeaturesScreen: BaseScreen {
+// This screen is currently unused. Given the purpose of UITestsFoundation is to provide developers
+// with an easy to use API to write tests, I'm going to keep the screen in the framework anyway, so
+// it'll be already available if someone needs to work with it or on it in the future.
+class BetaFeaturesScreen: ScreenObject {
 
     struct ElementStringIDs {
         static let enableProductsButton = "beta-features-products-cell"
     }
 
-    private let enableProductsButton = XCUIApplication().cells[ElementStringIDs.enableProductsButton]
+    // `expectedElement` is a `ScreenObject` utility to get the first element for the
```

Comment:
It looks like `from` is meant here?

```suggestion
    // `expectedElement` is a `ScreenObject` utility to get the first element from the
```